README.md

```diff
@@ -102,3 +102,31 @@ order to create everything with the right parameters. E.g.:
 There are a number of env vars this script looks at which are useful
 when running directly; check the script headers for details.
 
+## Throttling logs in Fluentd
+
+For projects that are especially verbose, an administrator can now throttle
+down the rate at which the logs are read in.  To tell Fluentd which projects
```

Comment:
I assume this is the basis for pertinent docs changes...

`s/now//` as all docs are in the present :)

Also can you clarify where/how exactly the rate is being affected? I.e. I think you mean logs in excess of the limit per project per node are delayed at fluentd (not entered into ES) until a later round of log gathering.